nodeServerless/Local CE SAM/event.json

```diff
@@ -0,0 +1,63 @@
+{
+  "body": "{}",
+  "resource": "/{proxy+}",
+  "path": "/path/to/resource",
+  "httpMethod": "GET",
+  "isBase64Encoded": false,
+  "queryStringParameters": {
```

Comment:
make these path params

Reply:
I need the hash key and the sort key to call the documentclient get operation. Should my path be "/cards/cardname-cardID/" then? Should I use a sort key generally?